installation/README.md

```diff
@@ -0,0 +1,31 @@
+# Compass installation and testing
+
+## Overview
+
+This document presents the deep dive into Compass installation and testing.
+
+## Installation
+
+To install the Compass, run:
+```bash
+./installation/scripts/run.sh
+```
+
+The `run.sh` scripts do the following:
```

Comment:
```
The `run.sh` scripts do the following:
1. Provision local Kubernetes cluster on Minikube adjusted for Kyma installation via `Kyma CLI`.
2. Install Kyma on the cluster with hardened list of components provided in `./installation/resources/installer-cr.yaml` file.  
3. Run `tiller-tls.sh` script to download and use Helm Client Certs created with Kyma installation.
4. Perform installation of `compass` chart.
```

This section could be moved to main README for local installation, but with modified 3rd point:
```
3. Downloads Helm client certificates created with Kyma installation.
```